portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Date;
+
+// A user-submitted comment
+public final class Comment{
```

Comment:
Just FYI, in more complicated use cases in industry, we'd usually use the Builder pattern. And there's some fancy tooling called AutoValue which makes it less wordy. 

https://howtodoinjava.com/design-patterns/creational/builder-pattern-in-java/